docs/contributors/github-labels.md

```diff
@@ -15,91 +15,96 @@ The labels in this list originated within Kubernetes at
-| ready-to-work                      | Indicates that an issue or PR has been sufficiently triaged and prioritized and is now ready to work. | Any |
-| size/L                             | Denotes a PR that changes 100-499 lines, ignoring generated files. | Any |
-| size/M                             | Denotes a PR that changes 30-99 lines, ignoring generated files.| Any |
-| size/S                             | Denotes a PR that changes 10-29 lines, ignoring generated files.| Any |
-| size/XL                            | Denotes a PR that changes 500+ lines, ignoring generated files.| Any |
-| size/XS                            | Denotes a PR that changes 0-9 lines, ignoring generated files.| Any |
-| triage/duplicate                   | Indicates an issue is a duplicate of other open issue. | Humans |
-| triage/needs-information           | Indicates an issue needs more information in order to work on it. | Humans |
-| triage/not-reproducible            | Indicates an issue can not be reproduced as described. | Humans |
-| triage/unresolved                  | Indicates an issue that can not or will not be resolved. | Humans |
+| api-review                           | Categorizes an issue or PR as actively needing an API review. | Any |
+| area/api                             | Issues or PRs related to an API | Any |
+| area/blog                            | Issues or PRs related to blog entries | Any |
+| area/build-release                   | Issues or PRs related to building and releasing | Any |
+| area/component/antctl                | Issues or PRs related to the command line interface component | Any |
+| area/component/agent                 | Issues or PRs related to the Agent component | Any |
+| area/component/cni                   | Issues or PRs related to the cni component | Any |
+| area/component/controller            | Issues or PRs related to the Controller component | Any |
+| area/component/flow-aggregator       | Issues or PRs related to the Flow Aggregator component | Any |
+| area/component/octant-plugin         | Issues or PRs related to the octant-plugin component | Any |
+| area/dependency                      | Issues or PRs related to dependency changes | Any |
+| area/endpoint/identity               | Issues or PRs related to endpoint identity | Any |
+| area/endpoint/selection              | Issues or PRs related to endpoint selection | Any |
+| area/endpoint/type                   | Issues or PRs related to endpoint type | Any |
+| area/flow-visualization              | Issues or PRs related to flow visualization support in Antrea | Any |
+| area/flow-visualization/aggregation  | Issues or PRs related to flow aggregation | Any |
+| area/flow-visualization/elk          | Issues or PRs related to the reference ELK configuration for flow visualization | Any |
+| area/flow-visualization/export       | Issues or PRs related to the Flow Exporter functions in the Agent | Any |
```

Comment:
As we are including exporting and aggregator functionality as the sub-areas, flow visualization could be slightly misleading, but at the same time, it can be considered as the umbrella feature. I think `flow-monitoring` can be considered more appropriate. No strong preference.

Reply:
I don't mind changing it to flow-monitoring. Or do you prefer flow-visibility since this is the term we use in the documentation: https://github.com/vmware-tanzu/antrea/blob/main/docs/network-flow-visibility.md#network-flow-visibility-in-antrea?

Reply:
done, thanks